### PR TITLE
Added option to hide disabled blocks

### PR DESCRIPTION
**Pull request by core23**
Added an option to hide disabled blocks when you are admin

### CODE REVIEW

**Comment by core23** [comment]
Ping @greg0ire @OskarStark

**Comment by greg0ire** [comment]
LGTM

File: Twig/Extension/PageExtension.php [hunk 1]
```diff
@@ -74,14 +79,16 @@ class PageExtension extends \Twig_Extension implements \Twig_Extension_InitRunti
...
      */
-    public function __construct(CmsManagerSelectorInterface $cmsManagerSelector, SiteSelectorInterface $siteSelector, RouterInterface $router, BlockHelper $blockHelper, HttpKernelExtension $httpKernelExtension)
+    public function __construct(CmsManagerSelectorInterface $cmsManagerSelector, SiteSelectorInterface $siteSelector, RouterInterface $router, BlockHelper $blockHelper, HttpKernelExtension $httpKernelExtension, $hideDisabledBlocks = false)
     {
         $this->cmsManagerSelector  = $cmsManagerSelector;
         $this->siteSelector        = $siteSelector;
         $this->router              = $router;
         $this->blockHelper         = $blockHelper;
         $this->httpKernelExtension = $httpKernelExtension;
+        $this->hideDisabledBlocks = $hideDisabledBlocks;
```
Comment (OskarStark) [inline Twig/Extension/PageExtension.php]: space missing

File: Twig/Extension/PageExtension.php [hunk 2]
```diff
@@ -74,14 +79,16 @@ class PageExtension extends \Twig_Extension implements \Twig_Extension_InitRunti
      * @param RouterInterface             $router              The Router
      * @param BlockHelper                 $blockHelper         The Block Helper
      * @param HttpKernelExtension         $httpKernelExtension
+     * @param bool                        $hideDisabledBlocks
      */
-    public function __construct(CmsManagerSelectorInterface $cmsManagerSelector, SiteSelectorInterface $siteSelector, RouterInterface $router, BlockHelper $blockHelper, HttpKernelExtension $httpKernelExtension)
+    public function __construct(CmsManagerSelectorInterface $cmsManagerSelector, SiteSelectorInterface $siteSelector, RouterInterface $router, BlockHelper $blockHelper, HttpKernelExtension $httpKernelExtension, $hideDisabledBlocks = false)
```
Comment (OskarStark) [inline Twig/Extension/PageExtension.php]: shall we linebreak this?

Comment (greg0ire) [inline Twig/Extension/PageExtension.php]: Definitely :)

**Comment by OskarStark** [comment]
ping @core23

**Comment by core23** [comment]
Done @OskarStark

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**